packages/lab/src/navbar/NavbarItem.tsx

```diff
@@ -83,6 +83,7 @@ export interface NavbarItemProps {
   label: ReactNode;
   icon?: ReactNode;
   badge?: ReactNode;
+  badgeIcon?: ReactNode;
```

Comment:
@SiriusCrain question: will it be icon that represent badge? or it will be just icon at the end of navbar item? if so, maybe we can call it `endIcon` as in some other components. *Not important issue.*

Reply:
@SiriusCrain Can we show `badge` for that?

Reply:
@SiriusCrain If count has failure then we will show warning icon as a `badge`?

Like this:

```jsx
<NavbarItem badge={failure ? <WarningIcon/> : <Badge/>} />
```

Reply:
We can, but current problem is that badge is part of this component, and I am not passing it from outside.
So in your suggestion I will need to update any usage of sidebar and extract badge from this lib, probably